src/frontend/src/views/Terms/index.tsx

```diff
@@ -478,9 +577,9 @@ export default function Terms(): JSX.Element {
             </NewTabLink>{" "}
             (email does not constitute notice)
             <br />
-          </P>,
-        ]}
-      />
+          </P>
+        </ListItem>
+      </List>
       <div style={{ height: "50px" }} />
```

Comment:
Want to remove this now because it's done via NarrowContainer, right?

Reply:
good catch